src/store/dapp-staking/actions.ts

```diff
@@ -614,11 +614,12 @@ const actions: ActionTree<State, StateInterface> = {
               };
             }
 
+            const hasStake = Number(balance.toString()) > 0;
```

Comment:
I see that some variables are using `parseInt` to convert from string to number, while this one uses `Number`. Is there a particular reason for this? Please make the code consistent or add comments

Reply:
I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/287/commits/e04aed3142393f05ef249b04ec578d737017c175) commit.

CC: @bobo-k2 @sirius651 
Let use `Number()` instead of `parseInt()` in this repo from now on :)